### DISCUSSION TITLE
Show all checks results even if no resources found

### DISCUSSION THREAD

**Discussion by crumbtifact** [prowler-cloud/prowler#2127] [Q&A]
Hi, I'd like to know whether it is possible to produce the report although there are no associated resources in the AWS account?

<img width="608" alt="image" src="https://user-images.githubusercontent.com/58942478/227477833-d56f7ccb-b77c-48f7-81c0-edb7bb6170eb.png">

The reason for this: the control description, remediation, etc. might be relevant for future references. I know it's possible to check the .json files 1 by 1 to get this information, but if prowler can produce the status to "Not Applicable" or "No Resources Detected" it might be very useful from users perspective.

**Comment by toniblyx** [comment]
Hi @crumbtifact thanks for let us know, we will give it a thought and let you know.

**Reply by scrapbird** [reply]
I'd like to add to this that not having this feature makes rules such as the GuardDuty enabled rules fairly useless as if there isn't a detector configured the check will not show up in the results so there is no failure raised.
